composer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "jeroen/mediawiki-github",
```

Comment:
I did not change this part. Sure it's nicer to have it be in extensions/GitHub than extensions/MediawikiGithub, but this is another issue, and should not cause any real problems